### ISSUE TITLE
Custom compile missing the animated intro

### ISSUE THREAD

**Issue by Mte90** [OpenRCT2/OpenRCT2#7557]
**OS:** Debian
**Version:** 0.1.2
**Commit/Build:** using v0.1.2 branch

I compiled on 2 different computers OpenRCT2 on debian without any issues but I am trying to understand why in the intro/menu there is only a green map without any animations like with the official build.
Usually there is like a map animated with music but with my version compiled is missing.

Maybe I am missing a command? on https://github.com/OpenRCT2/OpenRCT2/wiki/Building-OpenRCT2-on-Linux there aren't a list of commands for cmake (I added 2 packages that was missing for compiling on debian in that page).

**Comment by Gymnasiast** [comment]
These are called title sequences. Did you do `sudo make install`? It should download them.
You might also miss objects, which are also downloaded this way.

**Comment by Mte90** [comment]
This command is not explained in the wiki, I will try in any case.

**Comment by Mte90** [comment]
I get:
```
CMake Error at cmake_install.cmake:48 (file):
  file DOWNLOAD error: cannot create directory
  '/usr/local/share/openrct2/title' - Specify file by full path name and
  verify that you have directory creation and file write privileges.
```
And in the case I don't want to install it but only getting working?
I guess that `make install` require root permissions.

**Comment by AaronVanGeffen** [comment]
Pass `-DDOWNLOAD_TITLE_SEQUENCES` to cmake to download them.

**Comment by IntelOrca** [comment]
@Mte90 try `-DDOWNLOAD_TITLE_SEQUENCES=ON`

**Comment by Mte90** [comment]
Seems that with that is working, done a `make clean` and a new compile to see if they are included.

Later I will update the wiki page :-)

**Comment by Mte90** [comment]
Ah ok on the first run is downloading something with the window freezing:
```
object index out of date
Building object index (3083 items)
Finished building object index in 15.47 seconds.
Object conflict: '/home/mte90/Desktop/kde/RCT2_cd/ObjData/AE-LOOP.DAT'
               : '/home/mte90/Desktop/kde/RCT2_cd/ObjData/AE-LOOP2.DAT'
Object conflict: '/home/mte90/Desktop/kde/RCT2_cd/ObjData/INTLSMT1(1).DAT'
               : '/home/mte90/Desktop/kde/RCT2_cd/ObjData/INTLSMT1.DAT'
Object conflict: '/home/mte90/Desktop/kde/RCT2_cd/ObjData/XXWLBR07.DAT'
               : '/home/mte90/Desktop/kde/RCT2_cd/ObjData/XXWLBR08.DAT'
3 object conflicts found.
ERROR[/home/mte90/Desktop/kde/OpenRCT2/src/openrct2-ui/windows/Changelog.cpp:239 (window_changelog_read_file)]: Unable to read changelog.txt
```
But the title sequeece dropdown menu on settings is empty.

**Comment by AaronVanGeffen** [comment]
The changelog is read from the `doc` directory. In my build dir, I've symlinked to work around this:

```
aaron@izumo ~/Repositories/OpenRCT2/build (develop) $ ls -l doc
total 0
lrwxrwxrwx 1 aaron aaron 32 Mar 19 13:32 changelog.txt -> ../../distribution/changelog.txt
```

As for the object conflicts, you should just remove any duplicates as you see fit.

**Comment by Mte90** [comment]
Yes for changelog actually is not available in that folder so maybe this path need to be changed https://github.com/OpenRCT2/OpenRCT2/tree/develop/src/openrct2-ui/windows

**Comment by janisozaur** [comment]
1. There is absolutely no need to use `sudo` in getting openrct2. Ever. Period.
2. There is no need to manually pass any config options to get fully working openrct2 in a system with properly installed packages. No need for `-DDOWNLOAD_TITLE_SEQUENCES`
3. The compilation step is just that – it _compiles_ the project. To obtain all the external required files you need to install or package it properly, using nothing but the standard process, e.g. `$ DESTDIR=$(pwd)/out make install`

**Comment by AaronVanGeffen** [comment]
> Yes for changelog actually is not available in that folder so maybe this path need to be changed 

It's looking in quite a few folders, cf. [Platform::GetDocsPath](https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/openrct2/platform/Platform.Linux.cpp#L61)

**Comment by Mte90** [comment]
Thanks @janisozaur because as a normal user without specify a folder on make this will use `/usr/share` etc

Removing the duplicates object now on running I get:
```
object index out of date
Building object index (3077 items)
Finished building object index in 1.13 seconds.
```
In any case the title sequence is missing.
I am looking now about do a deb file for debian so there will be no issues on installing stuff in different folders.

**Comment by AaronVanGeffen** [comment]
> Alternatively, grab the [latest release](https://github.com/OpenRCT2/title-sequences/releases) and place them in the appropriate folder manually.

**Comment by Mte90** [comment]
I didn't find instructions for debhelper but as I can see looking on files is for ubuntu and not debian.

**Comment by janisozaur** [comment]
> is for ubuntu and not debian

What is for ubuntu and not debian?

**Comment by Mte90** [comment]
Sorry I didn'e explained pretty well.
The code that generate debian as I can see is for travis and openrct2 release package only for ubuntu.
I say only for ubuntu because they not works on debian that for few packages the versions required are not available, for that reason I compiled on my machine the game.

**Comment by Mte90** [comment]
Did you have a suggestion about what commands type to generate the package?
Looking on the code I didn't find them, I want to ask to debian developers if it is possible to include openrct2 in the repos.
